### ISSUE TITLE
M1.2: Train selectional preference model with tier-filtered embeddings

### ISSUE THREAD

**Issue by marctjones** [marctjones/klareco#622]
# Task

Train M1 selectional preference model using frozen tier-filtered root embeddings.

## Parent Epic

Part of **#616** (EPIC: Tier-Filtered Model Retraining)

## Model Architecture

~10M parameter model that scores plausibility of (subject, verb, object) triples:
```python
class M1SelectionalModel(nn.Module):
    def __init__(self, compositional_embedding):
        self.comp_emb = compositional_embedding  # Frozen root embeddings
        self.scorer = nn.Sequential(...)         # Learned scoring layers
```

## Training Configuration

- **Input**: (subject_root, verb_root, object_root) embeddings from frozen CompositionalEmbedding
- **Output**: Plausibility score [0, 1]
- **Loss**: Binary cross-entropy
- **Epochs**: 50 (early stopping patience=10)
- **Batch size**: 256
- **Learning rate**: 0.0001

## Training Data

From **#621**: `data/training/m1_selectional_tier5.jsonl`

## Output

File: `models/m1_selectional/best_model.pt`

## Acceptance Criteria

- [ ] Model uses frozen root embeddings (no gradient updates to roots)
- [ ] Training converges (loss decreases)
- [ ] Validation accuracy > 75%
- [ ] Model saved with checkpoint
- [ ] Training logs show no errors

## Command

```bash
./scripts/train_m1_selectional.sh --fresh
```

## Dependencies

- Blocked by: #621 (Generate M1 training data)
- Blocks: #623 (Test M1 quality)

**Comment by marctjones** [comment]
Closing as part of a 2026-09-08 backlog cleanup (see PLAN.md, DESIGN.md, CLAUDE.md's merge gate).

This issue predates the merge gate (adopted 2026-07-13) and the AST-native orchestrator rewrite. It describes work against the pre-orchestrator RAG pipeline or the learned model stack (M1/M2 embeddings, compositional embeddings, entity classifiers, neural rerankers) that was pruned from the repo -- klareco/embeddings/, klareco/models/, and klareco/summarization/ no longer exist. It has no milestone and was never benchmarked, matching exactly the failure mode CLAUDE.md documents (25 capability issues filed 2026-05-26, 8 implemented, none benchmarked).

Labeled archived-2026-09 so this batch is recoverable: gh issue list --state closed --label archived-2026-09. If this capability is still wanted, re-file it against the current architecture naming the benchmark metric and test the merge gate requires.
